src/ucp/core/ucp_ep.c

```diff
@@ -2252,21 +2252,22 @@ int ucp_ep_config_test_rndv_support(const ucp_ep_config_t *config)
            (config->key.cm_lane  != UCP_NULL_LANE);
 }
 
-unsigned ucp_ep_do_keepalive(ucp_ep_h ep)
+void ucp_ep_do_keepalive(ucp_ep_h ep, ucp_lane_map_t *lane_map)
 {
+    ucp_lane_map_t check_lanes = *lane_map;
     ucp_lane_index_t lane;
 
+    ucs_status_t status;
     if (!ucp_ep_keepalive_is_enabled(ep)) {
-        return 0;
+        return;
```

Comment:
let's remove ucp_ep_keepalive_is_enabled and put its code here (it's used only once)
and remove ` (ucp_ep_config(ep)->key.ep_check_map != 0)` from it since it's already done below